src/commands/dev/dev.js

```diff
@@ -501,7 +510,11 @@ const dev = async (options, command) => {
 
   log(`${NETLIFYDEVWARN} Setting up local development server`)
 
-  const devCommand = () => startFrameworkServer({ settings })
+  const devCommand = async () => {
+    const { ipVersion } = await startFrameworkServer({ settings })
+    // eslint-disable-next-line no-magic-numbers
```

Comment:
nit: I would actually create a constant for this.

Reply:
IMHO I think it's easier to understand in the context of `6 ? '::1' : '127.0.0.1'`

But I think it's personal preference just my 2 cents